docs/source/scaleVM.md

```diff
@@ -0,0 +1,165 @@
+| `KUBECONFIG_PATH` | Path to your kubeconfig (e.g. `/root/.kube/config`) |
+| `SCALE_NODES` | List of target worker nodes (e.g. `"[ 'worker-0', 'worker-1' ]"`) |
+| `THREADS_LIMIT` | Number of physical CPUs to use for parallelism |
+| `TIMEOUT` | Maximum duration (in seconds) for the run |
+| `RUN_ARTIFACTS_URL` | (Optional) Remote URL for saving logs |
+| `SAVE_ARTIFACTS_LOCAL` | If true, saves logs locally to `/tmp/benchmark-runner-run-artifacts` |
+
+---
+
+## 🧰 Fedora VM BootStorm (Ephemeral Storage)
+
+This mode loads Fedora VMs using ephemeral storage and verifies their readiness via the login page.
+
+### Fedora-Specific Parameters
+
+| Variable | Description                                                |
+|----------|------------------------------------------------------------|
+| `BOOTSTORM_SCALE` | Number of Fedora VMs per worker node                       |
+| `FEDORA_CONTAINER_DISK` | (Optional) Fedora .qcow2 image from quay.io (default: Fedora 37) |
+
+### Example Script
+
+```bash
+#!/bin/bash
+
+# Load Fedora 37 VMs including verification (verify login page)
+sudo podman run --rm -it \
+  -e WORKLOAD='bootstorm_vm' \
+  -e KUBEADMIN_PASSWORD="${KUBEADMIN_PASSWORD}" \
+  -e RUN_ARTIFACTS_URL="${RUN_ARTIFACTS_URL}" \
+  -e SAVE_ARTIFACTS_LOCAL=True \
+  -e SCALE="${BOOTSTORM_SCALE}" \
+  -e SCALE_NODES="${SCALE_NODES}" \
+  -e THREADS_LIMIT="${THREADS_LIMIT}" \
+  -e RUN_STRATEGY=True \
+  -e TIMEOUT="${TIMEOUT}" \
+  -e log_level="INFO" \
+  -v "${KUBECONFIG_PATH}:${KUBECONFIG_PATH}" \
+  -v /tmp/benchmark-runner-run-artifacts:/tmp/benchmark-runner-run-artifacts \
+  --privileged quay.io/ebattat/benchmark-runner:latest
+```
+
+## 🧰 Windows VM Bootstorm (ODF Storage)
+
+This mode loads Windows VMs using ODF persistent volumes and validates successful login.
+
+### Windows-Specific Parameters
+
+| Variable | Description                                                                                         |
+|----------|-----------------------------------------------------------------------------------------------------|
+| `WINDOWS_SCALE` | Number of Windows VMs per worker node                                                               |
+| `WINDOWS_URL` | URL to the Windows .qcow2 image shared via NGINX (e.g., `http://<host>:8083/windows.qcow2`) |
+
+### Example Script
+
+```bash
+#!/bin/bash
+
+# Load Windows VMs including verification (verify login home page)
+sudo podman run --rm -t \
+  -e WORKLOAD='windows_vm' \
+  -e KUBEADMIN_PASSWORD="${KUBEADMIN_PASSWORD}" \
+  -e SAVE_ARTIFACTS_LOCAL=True \
+  -e SCALE="${WINDOWS_SCALE}" \
+  -e SCALE_NODES="${SCALE_NODES}" \
+  -e THREADS_LIMIT="${THREADS_LIMIT}" \
+  -e WINDOWS_URL="${WINDOWS_URL}" \
+  -e RUN_STRATEGY=True \
+  -e TIMEOUT="${TIMEOUT}" \
+  -e log_level="INFO" \
+  -v "${KUBECONFIG_PATH}:${KUBECONFIG_PATH}" \
+  -v /tmp/benchmark-runner-run-artifacts:/tmp/benchmark-runner-run-artifacts \
+  --privileged quay.io/ebattat/benchmark-runner:latest
```

Comment:
Why does this need to be privileged?  That requires someone else to trust an image not under their control.

Reply:
I can try to run it w/o it.

Reply:
Please do, that would be greatly preferable.